src/recipe.yaml

```
@@ -69,16 +70,20 @@ requirements:
     - epiweeks
     - git
     - google-cloud-storage
-    - gzip
+    # gzip is not yet available for arm64 on conda-forge
+    # https://github.com/conda-forge/gzip-feedstock/issues/8
+    # however gzip is already installed on stock macOS
+    # We don't seem to support selectors, so temporarily disable gzip globally
+    # - gzip  # [not arm64]
```

```suggestion
    # TODO: Remove selector once gzip is available for arm64
    # i.e. once https://github.com/conda-forge/gzip-feedstock/pull/9 is merged
    - sel(not arm64):
      - gzip
```

Thanks @tsibley - I haven't worked with boa before, that pointer is very useful

I wasn't necessarily suggesting for us to remove gzip for all environments, just a temporary workaround for osx-arm64 to unblock tests here until the PR in the feedstock is merged.

---

This works!